pisek/jobs/status.py

```diff
@@ -29,7 +28,10 @@ def _bar(self, msg: str, part: int, full: int, color: str = "cyan") -> str:
         msg = pad(msg, MSG_LEN)
         progress_msg = f"  ({part}/{full})"
 
-        bar_len = min(terminal_width - len(msg) - len(progress_msg), MAX_BAR_LEN)
+        TARGET_BAR_WIDTH = TARGET_LINE_WIDTH - MSG_LEN - 11
```

Comment:
It's a bit weird to declare a constant inside of a function. (See also [PEP 8](https://peps.python.org/pep-0008/#constants).)